internal/servers/controller/handlers/authmethods/authmethod_service.go

```diff
@@ -71,8 +71,6 @@ var (
 			action.Read,
 			action.Update,
 			action.Delete,
-			action.ChangePassword,
```

Comment:
I know it's not really in scope for this PR... would you mind adding better coverage for some of these functions?  There's a couple that appear to have no coverage.  ty!